modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -158,321 +102,259 @@ INDEXED_EXTRACTIONS = JSON
 
-|`sslSecurityProtocol`
-|String
-|SSL Security Protocol to use in the https connection. Possible values: TLSv1_2, TLSv1_1, TLSv1, SSLv3.
-|Optional
-|TLSv1_2
 
-|`splunkIndexName`
-|String
-|Splunk index name where all the events must be sent. If the user has the rights,
-and the index doesn't exist, then the internal handler will create it.
-|Optional
-|main
+. If you want, click *Show more options* to specify formatting properties for the sent data:
++
+.The arrow shows the expanded options in the Splunk configuration window.
+image::hybrid-export-splunk-configure-advanced.png[Expanded options in the Splunk configuration window]
++
+See <<Splunk Configuration Options>>.
++
+The *SSL Protocol* option applies only to the REST API data input type.
++
+[IMPORTANT]
+If you set values for the Splunk index, Splunk source, and Splunk source type when you registered your data input in Splunk, any values you specify in this window overwrites those values.
 
-|`splunkSource`
-|String
-|The source used on the events sent to Splunk.
-|Optional
-|mule
+. Click *Apply* to save your changes.
 
-|`splunkSourceType`
-|String
-|The sourcetype used on the events sent to Splunk.
-|Optional
-|mule
+=== Splunk Configuration Options
 
-|`dateFormatPattern`
-|String
-|Date format used to format the timestamp.
-|Optional
-|yyyy-MM-dd'T'HH:mm:ssSZ
+Splunk integration includes the following configuration options:
 
+[cols="20,30,20"]
 |===
+|Field|Description|Default Value
 
-.Splunk Internal Handler Minimum Configuration Example
-[source,yaml]
-....
----
-  mule.agent.gw.http.handler.splunk:
-    host: 192.168.61.131
-    user: admin
-    pass: test
-....
-
-=== Rest API
+|Username
+(Required)
+|Username to connect to Splunk
+|
 
-This feature requires the Runtime Manager agent 1.2.0 version or later.
+|Password 
+(Required)
+|Password associated with the Splunk username 
+|
 
-. Select the server whose information you want to send out.
-. In the menu on the right, click *Manage Server* to access the server's settings.
-+
-image::sending-data-from-arm-to-external-monitoring-software-manage-server.png[]
+|host
+(Required)
+|IP address or hostname of the server where Splunk is running
+|
 
-. Select the *Plugins* tab:
-+
-image::sending-data-from-arm-to-external-monitoring-software-plugins.png[]
+|port
+|Splunk management port
+|8089
 
-. Select the kind of information that you want to send out in the *Level* dropdown menu.
-+
-image::sending-data-from-arm-to-external-monitoring-software-level.png[]
+|protocol 
+a|Protocol for the connection to the Splunk management port
 
-. On the *Event Tracking* region, activate the *Splunk* switch. A pop-up menu opens where you provide your Splunk user and password data, as well as the host and port for the connection.
-+
-image::sending-data-from-arm-to-external-monitoring-software-splunk.png[]
+Possible values are http, https, and tcp.
```

Comment:
This lowercasing is what made me think twice about the tagging....

Reply:
Punted for now per Kim